app/graphql/mutations/favorite_street_art.rb

```diff
@@ -0,0 +1,12 @@
+class Mutations::FavoriteStreetArt < Mutations::BaseMutation
+  argument :favorite, Boolean, required: true
+  argument :street_art_id, Integer, required: true
+
+  type Types::StreetArtType
+
+  def resolve(favorite: ,street_art_id:)
+    art = StreetArt.find(street_art_id)
+    art.update(favorite: favorite)
+    art.reload
```

Comment:
Thanks for the `.reload` tip, Stella!

Reply:
![](https://media1.giphy.com/media/AuQvTyPtkZu8w/giphy.gif?cid=5a38a5a2adxc5s75pn381xbq6xkn699hwh9lwsc3le4f89io&rid=giphy.gif)